addons/binding/org.openhab.binding.oceanic/build.properties

```diff
@@ -3,4 +3,4 @@ output.. = target/classes
 bin.includes = META-INF/,\
                .,\
                OSGI-INF/,\
-               ESH-INF/
\ No newline at end of file
+               ESH-INF/           
```

Comment:
No, it doesn't. But the SCATool tells me that a new line is missing. Will make the quick change